lib/middleware/campaigns/single/bot-config-parse.js

```diff
@@ -6,19 +6,18 @@ module.exports = function parseBotConfig() {
   return (req, res) => {
     try {
       req.campaign.botConfig = {
-        postType: helpers.botConfig.parsePostTypeFromBotConfig(req.botConfig),
+        postType: helpers.botConfig.getPostTypeFromBotConfig(req.botConfig),
       };
-      const templateNames = helpers.botConfig.getTemplateNames();
-      const templatesObj = {};
+      const templates = helpers.botConfig.getTemplatesFromBotConfig(req.botConfig);
```

Comment:
Minor: I wonder if you could pass the `req.campaign` here so you could move the looping to get the rendered templates into  `getTemplatesFromBotConfig`.